client/quantum_serverless/core/__init__.py

```diff
@@ -39,7 +39,6 @@
     RayJobClient
     save_result
     QiskitPattern
-    Program
```

Comment:
This is causing Sphinx errors for me since it is no longer imported in this file, not sure how main branch is passing sphinx build with no warnings?